submissions/sprint_b/helpers.py

```diff
@@ -0,0 +1,180 @@
+import os
+import json
+import random
+import unittest
+
+class FileManager:
+    """Handle local File System IO"""
+
+    @staticmethod
+    def get_extension(path):
+        """Get extension from file path"""
+        return os.path.splitext(path)[-1][1:]
+
+    @staticmethod
+    def read_json(path, mode="r", *args, **kwargs):
+        with open(path, mode=mode, *args, **kwargs) as handle:
+            return json.load(handle)
+
+
+class Vocabulary:
+    """Standardize vocabulary representation from multiple sources."""
+
+    files = FileManager()
+
+    @classmethod
+    def from_file(cls, path, *args, **kwargs):
+        extension = cls.files.get_extension(path)
+        representation = cls.strategies(extension)(path, *args, **kwargs)
+
+        return representation
+
+    @classmethod
+    def from_json(cls, path, fields=True, *args, **kwargs):
+        data = cls.files.read_json(path, *args, **kwargs)
+        if fields:
+            representation = (data, data.keys())
+        else:
+            representation = data
+        print representation
+        return representation
+
+    @classmethod
+    def strategies(cls, file_extension, intent='read'):
+        input_strategies = {"json": cls.from_json}
+        if intent is "read":
+            return input_strategies[file_extension]
+
+class EpithetGenerator:
+    """Pull from the reasources folder to generate the vocabulary and then use the
+    vocabulary to generate epithets"""
+
+
+    @classmethod
+    def vocab_generator(cls):
+        file_manager = FileManager()
+        vocabulary = Vocabulary()
+        resources = ['/Users/Ben/Documents/Kenzie/Code/Activities/PYTHON/backend-epithet-generator/resources/data.json']
```

Comment:
Use of absolute paths to files on your development environment is causing routes to fail. Use relative paths to the repo's resources directory instead.